chalicelib/action.py

```diff
@@ -212,21 +231,22 @@ def _edit_action(self):
 
         event_to_edit = None
         try:
-            event_to_edit = json.loads(self._get_events(date_str=date_input))
+            event_to_edit = read_event(
+                url=self.config["backend_url"], user_id=self.user_id, date=date_input
+            )
         except json.decoder.JSONDecodeError as error:
```

Comment:
Hm this exception is probably not going to happen anymore since we don't try to decode json here.
Should I remove the whole try/except?

Reply:
We should check for status 200 on api call

`if event_to_edit.status_code != 200` handle api problem

Reply:
Yeah but does our API ever return anything else than 200? Probably 500

Reply:
Changed so it checks the `status_code` instead